### PR TITLE
chore: support development with pnpm

### PR DESCRIPTION
**Pull request by sidvishnoi**
pnpm uses Node's resolve algorithm, so some packages failed to resolve (npm flattens dependency tree, so all resolved fine there).

This PR is allowing development with pnpm for those who want to use it. npm will continue to work fine. If someone wants to use pnpm locally, they can:
```bash
rm -rf node_modules
pnpm import # imports package-lock.json and creates a pnpm-lock.yaml
pnpm install
# and everything works fine as is
```

When updating a dependency in `package-lock.json`, we can:
```bash
npm i --package-lock-only # this will not touch node_modules
pnpm import && pnpm install
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "rollup-plugin-minify-html-literals": "^1.2.6",
     "rollup-plugin-terser": "^7.0.2",
     "serve": "^13.0.2",
+    "serve-handler": "^6.1.3",
```
Comment (sidvishnoi) [inline package.json:60]: We would've found this failing when we removed the last package that has `serve-handler` as a dependency.

**Comment by marcoscaceres** [comment]
neat! but didn't pnpm ship with Node 17? We can't use 17 yet (at least, I don't have pnpm installed?).

**Comment by sidvishnoi** [comment]
With Node 16, if you run `corepack enable`, it'll enable pnpm (which is installed already but hidden).

File: package.json (hunk 1 above)
Comment (marcoscaceres) [reply package.json:60]: I don't even remember what it does...

Comment (sidvishnoi) [reply package.json:60]: It's used in dev-server

**Pull request merged**